backend/notification-service/src/main/resources/application.yml

```diff
@@ -0,0 +1,34 @@
+server:
+  port: 777
+  tomcat:
+    uri-encoding: UTF-8
+spring:
+  application:
+    name: Notification service
+  mail:
+    host: ${SMTP_HOST}
+    username: ${SMTP_USER}
+    password: ${SMTP_PASS}
```

Comment:
Uses environment variables. They'd be taken from .env file in notification-service folder